src/app/components/Curation/CurationGrid/index.styles.ts

```diff
@@ -34,7 +34,7 @@ const styles = {
           display: 'inline-block',
           verticalAlign: 'top',
         },
-        'div div:nth-child(2)': {
+        'div div:last-child': {
```

Comment:
Shouldn't affect anything else, but the added in blurred image makes it so that there are now 3 children